Jhs tw 1819

JHStoops opened the pull request:
![image](https://user-images.githubusercontent.com/5678259/51708186-79510900-1fe0-11e9-8dc8-dd8c396b0d0b.png)

Enabled ellipsis on overflow.

justincy commented on src/fs-person.html:
```diff
@@ -315,7 +341,7 @@
   `;
 
-  FS._registerTranslations(/* LANG CODE */);
+  FS._registerTranslations({"de":{"_LRM_PKTAG631":"fs-person_1_146 - FULL-FILE","fs.shared.fsPerson.FEMALE":"weiblich","fs.shared.fsPerson.MALE":"männlich","fs.shared.fsPerson.UNKNOWN":"Geschlecht unbekannt","fs.shared.fsPerson.UNKNOWN_NAME":"[Unbekannter Name]"},"en":{"fs.shared.fsPerson.UNKNOWN_NAME":"[Unknown Name]","fs.shared.fsPerson.MALE":"Male","fs.shared.fsPerson.FEMALE":"Female","fs.shar ...
```
I don't believe this is supposed to be here. It looks like code was copied from the file `fs-person.html` which is built based on this file.

justincy commented:
So when looking at the diff, your 1-line change is shown in `fs-person.html` while porting Jason's changes into src is represented in `src/fs-person.html`.

JHStoops commented:
right, I ported over Jason's code and changed the one line in his code, then built it so it reflected in the fs-person.html file as that single line in src/fs-person.html. 
Also, out of habit I added the two files to the .gitignore file; I just realized you might not want them there. Do you want me to revert that back?

justincy commented:
You can keep the `package-lock.json` entry but you ought to remove `analysis.json` since it's in the repo already.

JHStoops commented:
I just took out analysis.json from .gitignore and pushed.

justincy commented:
Good work!
